lib/build.command.js

```diff
@@ -14,25 +13,31 @@ module.exports = function (Aquifer) {
     return;
   }
 
-
   /**
    * Define the 'build' command.
    */
   Aquifer.cli.command('build')
   .description('Builds a Drupal site in the /builds/work directory.')
-  .action(function () {
+  .option('-l, --lock <file.lock>', 'Override the lock file defined in settings for the build or enter "false" to skip lock file and build from make file. Existing lock file will be preserved.')
+  .option('-m, --make <file.make>', 'Override the make file defined in settings for the build. Lock file will be rewritten unless you also set --lock to "false".')
```

Comment:
In aquifer.json, we standardize on one makefile, and I think that might be best practice. If changes need to be made to the makefile, I think it might be best to just make changes to the project's make file and re-test.